expression/builtin_time.go

```diff
@@ -53,8 +53,12 @@ const ( // GET_FORMAT location.
 // DurationPattern determine whether to match the format of duration.
 var DurationPattern = regexp.MustCompile(`^(|[-]?)(|\d{1,2}\s)(\d{2,3}:\d{2}:\d{2}|\d{1,2}:\d{2}|\d{1,6})(|\.\d*)$`)
 
+// DatePattern determine whether to match the format of date.
+var DatePattern = regexp.MustCompile(`^\s*((0*\d{1,4}([^\d]0*\d{1,2}){2})|(\d{2,4}(\d{2}){2}))\s*$`)
```

Comment:
no need to export this variable, so does `DurationPattern`

Reply:
ok